### DISCUSSION TITLE
Unit models update

### DISCUSSION THREAD

**Discussion by yvonneadw** [IDAES/idaes-pse#1174] [Developer News Bytes]
Hello, I'm trying to use IDAES to simulate a water tank, steady state and dynamic state. Is the latest version of the syntax available yet? Because I could not find it everywhere. For dynamic state, I was required by IDAES to provide time units but I could not find the proper syntax for time unit either. Any help is appreciated, thank you!

**Comment by andrewlee94** [comment]
@yvonneadw If you are referring to the release of IDAES v2.0 then yes it is available, however the syntax for time units has not changed as part of that. In order to set the time units, you need to provide a Pyomo unit object similar to what is shown below:

```
m.fs = FlowsheetBlock(time_units=pyomo.units.s)
```
